config/sync/language/da/block.block.claro_breadcrumbs.yml

```diff
@@ -1,2 +1,2 @@
 settings:
-  label: Brødkrummer
+  label: Breadcrumbs
```

Comment:
Is it due to how config_translation and/or config_translation_po work that we need English labels in the `da` folder?

Reply:
This here is a mistake, we should aim to have all strings translated in the da folder, and we should never trnaslate a danish string from config folder into an english string in da folder.
However.
Regarding config translations in general:
The "Config Translation Po" - module will look through db config and export all strings found to .po file. It will look for a danish translation and use it, but if none is found it will use the english string instead. So all config is mapped to a string in the .po file. Many will be the same though. When importing config translations the config is overwriten in the database.
Thus, the config/languages/da folder must match the translations/config/*.po file(s) . This is done by exporting config after importing changes in the .po files.